kafka-connect-transforms/src/main/java/io/tabular/iceberg/connect/transforms/CopyValue.java

```diff
@@ -81,24 +81,24 @@ public R apply(R record) {
   }
 
   private R applySchemaless(R record) {
-    final Map<String, Object> value = Requirements.requireMap(record.value(), "copy value");
+    Map<String, Object> value = Requirements.requireMap(record.value(), "copy value");
```

Comment:
Is removing `final` here necessary?  Seems like an unrelated change.

Reply:
Yeah, I was cleaning some things up I noticed, this is unrelated (should have mentioned that)